scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/InputSizeReducerEstimator.scala

```diff
@@ -28,30 +28,26 @@ class InputSizeReducerEstimator extends ReducerEstimator {
    * Figure out the total size of the input to the current step and set the number
    * of reducers using the "bytesPerReducer" configuration parameter.
    */
-  override def estimateReducers(info: FlowStrategyInfo): Option[Int] =
-    Common.inputSizes(info.step) match {
-      case Nil =>
-        LOG.warn("InputSizeReducerEstimator unable to estimate reducers; " +
-          "cannot compute size of:\n - " +
-          Common.unrollTaps(info.step).filterNot(_.isInstanceOf[Hfs]).mkString("\n - "))
-        None
```

Comment:
I think we still need the Nil case. There can be steps with non-Hfs source taps only. How about keeping the `unrollTaps` implementation for logging purposes here and using the new `inputSizes` for computing input sizes?